email_tracking/app.rb

```diff
@@ -0,0 +1,54 @@
+require 'sinatra/base'
+require 'easypost'
+require 'sendgrid-ruby'
+require 'tilt/erb'
+require 'pry'
+require 'dotenv'
+
+class App < Sinatra::Base
+  set :show_exceptions, :after_handler
+
+  configure :development, :test do
+    Dotenv.load
+  end
+
+  configure do
+    EasyPost.api_key = ENV['EASYPOST_API_KEY']
+    set :sendgrid, SendGrid::API.new(api_key: ENV['SENDGRID_API_KEY'])
+  end
+
+ post '/easypost-webhook' do
+    request.body.rewind
+    request_string = request.body.read
+    response = JSON.parse(request_string)
```

Comment:
oh.. damn.. sorry.. I was testing with a string before.. again "FORGOT TO REMOVE" error il remove the rewind